RockWeb/Themes/NewSpring/assets/Lava/UI/locations-hero.lava

```diff
@@ -1,44 +1,43 @@
-{% assign campusSlug = 'Global' | PageParameter:'CampusSlug' | Replace:'-',' ' | Uppercase %}
-{% campus where:'Name == "{{ campusSlug }}"' iterator:'campuses' %}
-{% for campus in campuses %}
+{% assign campusSlug = 'Global' | PageParameter:'CampusSlug' | Replace:'-',' ' | Capitalize %}
+{% assign campuses = 'campuses' | PersistedDataset %}
+{% assign campus = campuses | Where:'Name', campusSlug | First %}
```

Comment:
You can actually combine lines 1 & 2:
`{% assign campus = 'campuses' | PersistedDataset | Where:'Name', campusSlug | First %}`